Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -1351,14 +1368,41 @@ codeunit 4037 "Helper Functions"
             Codeunit.Run(Codeunit::"Stat. Acc. Post. Batch", StatisticalAccJournalLine);
     end;
 
-    local procedure PostItemBatch(ItemJournalBatch: Record "Item Journal Batch")
+    local procedure SafePostStatisticalAccBatch(JournalBatchName: Code[10])
+    var
+        StatisticalAccJournalLine: Record "Statistical Acc. Journal Line";
+    begin
+        StatisticalAccJournalLine.SetRange("Journal Batch Name", JournalBatchName);
+        if StatisticalAccJournalLine.FindFirst() then begin
```

Comment:
Same comment here, we need to do a FindSet, if the codeunit does this, then we can make a check on the IsEmpty